app/routes/about.tsx

```diff
@@ -27,54 +23,64 @@ export default function DashboardAbout() {
     <div className="page">
       <div className="about-us-container ">
         <h1 className="sm:text-8xl text-6xl ">About Us</h1>
-        <div className="aboutus-container flex flex-col lg:flex-row justify-center items-center md:gap-20 sm:gap-24 gap-6 px-20 mb-20 xl:my-44">
-          <div className="aboutus-text font-bold my-10 md:text-2xl 2xl:text-3xl text-lg -z-10">
+        <div className="aboutus-container flex flex-col-reverse lg:flex-row justify-center items-center md:gap-20 sm:gap-24 gap-6 md:px-20 px-10">
+          <div className="aboutus-text font-bold mt-5 mb-5 md:my-0 md:text-xl 2xl:text-2xl text-lg -z-10">
             We are Concordia's gamedev community and we focus on the technical
             and development aspects of video games in all disciplines, such as,
             programming, art, game design, storytelling, sound engineering and
             more.
           </div>
-          <div className="blob w-full flex items-center sm:my-32 lg:my-28 -z-20 md:max-lg:mt-60 mt-28 sm:mb-0 mb-16">
-            <img
+          <div className="blob w-full flex items-center md:mt-0 mt-10 md:p-10">
```

Comment:
The size of the controller seems a little too big for some screen sizes (I think md and xl), which makes for a lot of scrolling before we get to the team section.

![image](https://github.com/user-attachments/assets/e3a36aa9-cc90-44d9-b95a-cacb383f683e)
![image](https://github.com/user-attachments/assets/d853f19d-b997-4106-8513-8da958fc3cd1)